git/README.md

```diff
@@ -181,3 +181,71 @@ There are 4 additional parameters in addition to the ones mentioned above for th
 [git-tree]: https://git-scm.com/book/en/v2/Git-Internals-Git-Objects
 [tide]: https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md
 [zuul-ci]: https://zuul-ci.org/docs/zuul/discussion/gating.html
+
+
+## `git-rebase`
+
+This [task](../git/git-rebase.yaml) will `rebase` the branch based
+on the user input. Before rebase, if `squashing` of the commits is required,
+then it can be done by providing the `squash count` i.e number of commits to squash.
+
+### Workspaces
+
+* **source**: A workspace that contains the fetched the git repository.
+
+### Parameters
+
+* **SQUASH_REQUIRED**: A boolean value which specifies if 
+squashing commits is required or not before rebase (_default:_ `true`).
+* **SQUASH_COUNT**: Number of commits to squash in the branch.
+(_only required if squashing needs to be performed_).
```

Comment:
Can we use `SQUASH_COUNT` itself to decide whether SQUASH is required or not ?
so that we can avoid `SQUASH_REQUIRED` arg

Reply:
Yes, thanks for the suggestion :)